apps/test/unit/applab/ImportScreensDialogTest.js

```diff
@@ -2,7 +2,6 @@
 import React from 'react';
 import sinon from 'sinon';
 import {shallow, mount} from 'enzyme';
-import MultiCheckboxSelector from '@cdo/apps/templates/MultiCheckboxSelector';
 import {expect} from '../../util/configuredChai';
```

Comment:
would it be worth it to configure an alternate version of configuredChai (perhaps named reconfiguredChai) which does not include chai-enzyme? That way, you could change the import to pull in `reconfiguredChai` instead of `configuredChai` to prove that we're no longer relying on chai-enzyme in this file. It could also speed up the debugging process, since it would allow you to remove the chai-enzyme dependency and then rely on test failures to catch any errors you may have missed.

Reply:
Good call. I'll make that change.